packages/plugins/puppetlabs-stdlib/puppetlabs-stdlib.spec

```
@@ -0,0 +1,78 @@
+Name:		puppetlabs-stdlib
+Version:	5.2.0
+Release:	1%{?dist}
+Summary:	Puppet Labs Standard Library
+License:	ASL 2.0
+URL:		https://github.com/puppetlabs/puppetlabs-stdlib
+Source0:	https://forge.puppet.com/v3/files/%{name}-%{version}.tar.gz
+BuildArch:	noarch
+Requires:	puppet >= 5.5.10
+Requires:	puppet < 7.0.0
+
+%description
+Puppet Labs Standard Library module.
+
+%prep
+%setup -qn %{name}-%{version}
+
+%build
+
+%install
+mkdir -p %{buildroot}/%{_datadir}/puppet/modules/stdlib
+cp -rp lib/ %{buildroot}/%{_datadir}/puppet/modules/stdlib/lib
+cp -rp spec/ %{buildroot}/%{_datadir}/puppet/modules/stdlib/spec
+cp -rp manifests/ %{buildroot}/%{_datadir}/puppet/modules/stdlib/manifests
+cp -rp types/ %{buildroot}/%{_datadir}/puppet/modules/stdlib/types
+cp -p metadata.json %{buildroot}/%{_datadir}/puppet/modules/stdlib/
```

You can also consider combining these:
```suggestion
cp -rp lib/ manifests/ types/ metadata.json %{buildroot}%{_datadir}/puppet/modules/stdlib/
```